mrmustard/lab/representations/data/array_data.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+import numpy as np
+from typing import List, Union
+from mrmustard.lab.representations.data.data import Data
+from mrmustard.math import Math
+from mrmustard.typing import Scalar, Vector
+
+math = Math()
+
+class ArrayData(Data):
+    """ Contains array-like data for certain Representation objects.
+
+    Args:
+        array : data to be contained in the class
+    """
+
+    def __init__(self, array:Vector) -> None:
+        self.array = array
+        self.cutoffs = self.array.shape
+
+    
+    @property
+    def cutoffs(self) -> Union[int, List[int]]:
+        """ The cutoffs for the Fock representation. """
+        return self.array.shape()
+
+
+    def __neg__(self) -> Data:
```

Comment:
I realized neg is the implementation of `not` so we probably don't need it

Reply:
Good observation. Thanks. Let me remove it.